syntax = "proto3";

package phone_book_serialize;

message Date {
    uint32 year = 1;
    uint32 month = 2;
    uint32 day = 3;
}

message Contact {
    string name = 1;
    Date birthday = 2;
    repeated string phone_number = 3;
}

message ContactList {
    repeated Contact contact = 1;
}